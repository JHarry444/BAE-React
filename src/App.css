.App {
  text-align: center;
}

.App input {
  display: block;
  margin: auto;
  margin-top: 5px;
}

