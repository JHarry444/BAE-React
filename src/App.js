import { useState } from 'react';
import './App.css'; 
import Container from './lifting-state/Container';
import TrainerContainer from './trainers/TrainerContainer';

function App() {

  return (
    <div className="App">
      <TrainerContainer/>
      {/* <Container/> */}
    </div>
  );
}

export default App;
