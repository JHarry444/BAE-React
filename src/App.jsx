import './App.css';
import 'bootstrap/dist/css/bootstrap.min.css';
import { BrowserRouter as Router, Link, Route } from 'react-router-dom';
import { Nav } from 'react-bootstrap';
import TrainerContainer from './trainers/TrainerContainer';
import TrainerForm from './trainers/TrainerForm';
import IndividualTrainer from './trainers/IndividualTrainer';

function App() {
  return (
    <div className="App">
      <Router>
        <Nav fill variant="tabs">
          {/* routing Links with Bootstrap styling */}
          <Link to="/" className="nav-link">Home</Link>
          <Link to="/away" className="nav-link">Away</Link>
          <Link to="/trainers/create" className="nav-link">CREATE Trainers</Link>
          <Link to="/trainers/read" className="nav-link">GET Trainers</Link>
        </Nav>

        <br />
        <Route exact path="/">
          <p>Home</p>
        </Route>
        <Route path="/away">
          <p>Away</p>
        </Route>
        <Route path="/trainers/create">
          <TrainerForm />
        </Route>
        <Route exact path="/trainers/read">
          <TrainerContainer />
        </Route>
        <Route path="/trainers/read/:id">
          <IndividualTrainer />
        </Route>
      </Router>
    </div>
  );
}

export default App;
