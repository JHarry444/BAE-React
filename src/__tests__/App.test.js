import { render } from '@testing-library/react';
import App from '../App';

test('matches snapshot', () => {
  expect(render(<App />)).toMatchSnapshot();
});

test('equals 2', () => {
  expect(1 + 1).toEqual(2);
})