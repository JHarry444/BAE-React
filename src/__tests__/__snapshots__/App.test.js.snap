// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="nav nav-tabs nav-fill"
        >
          <a
            class="nav-link"
            href="/"
          >
            Home
          </a>
          <a
            class="nav-link"
            href="/away"
          >
            Away
          </a>
          <a
            class="nav-link"
            href="/trainers/create"
          >
            CREATE Trainers
          </a>
          <a
            class="nav-link"
            href="/trainers/read"
          >
            GET Trainers
          </a>
        </div>
        <br />
        <p>
          Home
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="nav nav-tabs nav-fill"
      >
        <a
          class="nav-link"
          href="/"
        >
          Home
        </a>
        <a
          class="nav-link"
          href="/away"
        >
          Away
        </a>
        <a
          class="nav-link"
          href="/trainers/create"
        >
          CREATE Trainers
        </a>
        <a
          class="nav-link"
          href="/trainers/read"
        >
          GET Trainers
        </a>
      </div>
      <br />
      <p>
        Home
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
